Tests/TensorFlowTests/InitializerTests.swift

```diff
@@ -71,13 +71,32 @@ final class InitializerTests: XCTestCase {
         XCTAssertEqual(ShapedArray(shape: [2, 2], scalars: [1, 0, 1, 0]), i8s.array)
     }
 
+    func testOrthogonalShapesValues() {
+        for shape in [[10, 10], [10, 9, 8], [100, 5, 5], [50, 40], [3, 3, 32, 64]]{
+            // Check the shape
```

Comment:
End each comment with a period.
```suggestion
            // Check the shape.
```